dapr/proto/runtime/v1/dapr.proto

```diff
@@ -380,7 +379,7 @@ message InvokeBindingResponse {
 // GetSecretRequest is the message to get secret from secret store.
 message GetSecretRequest {
   // The name of secret store.
-  string store_name = 1;
+  string store_name = 1 [json_name = "storeName"];
```

Comment:
By using the `json_name` annotation in protos, it's possible to define what the key in the JSON objects will be, as in this example

> Although in this specific case, this property is set by the `inModifier` method and read from the URL, so it doesn't really matter here.